tests/routerTest.php

```diff
@@ -62,6 +62,14 @@ public function testRouterInvokedSuccessfully()
 
         $this->expectOutputString('Invoked!');
     }
+
+    public function testCloudStorageStreamWrapperNotRegisteredByDefault()
+    {
+        $wrappers = stream_get_wrappers();
+        require 'router.php';
```

Comment:
Are suggested packages automatically installed with `require-dev` now?

Reply:
No, they aren't, but this test is checking to make sure the stream wrapper _isn't_ registered (because `google/cloud-storage` isn't a dep)

Reply:
This test can probably be removed since we have a better test in [`testGcsIsNotRegistered`](https://github.com/GoogleCloudPlatform/functions-framework-php/pull/75/files#diff-bdbbedd949c61ad749636e3450179483665e775c0d49e088ca944c45b1f16374R98) below